src/main/java/io/beekeeper/pingdom/resources/HealthCheckDetails.java

```diff
@@ -2,13 +2,33 @@
 
 public class HealthCheckDetails {
     public static class Severity {
-        public static final String KEY = Severity.class.getName() + ".severity";
-        public static final String VALUE_HIGH = "high";
-        public static final String VALUE_LOW = "low";
-        public static final String[] VALUES = { VALUE_HIGH, VALUE_LOW };
+        private static final String KEY = Severity.class.getName() + ".severity";
```

Comment:
Sorry, but this seems fishy: Why should you not be able to have a public static final? Is it because it's an inner class?

Reply:
I think [1] clearly describes problem and the final solution.
Thanks for pointing out @uvifian . Now it looks better.

[1]: https://stackoverflow.com/questions/16646319/malicious-code-vulnerability-field-should-be-package-protected

Reply:
Soooo much better! Thanks @kasatch

Reply:
Well, thank you Urs! :)